_episodes/06-branching.md

```diff
@@ -131,158 +145,217 @@ branch.
 
 ~~~
 $ git branch
-* cleanup-plan
   master
+* title-caps
 ~~~
 {: .bash}
 
+>Tip: You can create and checkout a branch at the same time using `git checkout`
```

Comment:
Is this what the `{: .callout}` tag is for?

Reply:
I have no idea.